clients/fossid-webapp/src/main/kotlin/EntityResponseBody.kt

```diff
@@ -23,7 +23,7 @@ class EntityResponseBody<T>(
     val operation: String? = null,
     val status: Int? = null,
     val message: String? = null,
-    val messageParameters: Array<String>? = null,
+    val messageParameters: Map<String, String> = emptyMap(),
```

Comment:
Commit message: Please mention in which version of FossID this has changed in the API. (And why it's not necessary to support the older API anymore.)

Reply:
> Commit message: Please mention in which version of FossID this has changed in the API. (And why it's not necessary to support the older API anymore.)

There is a reason I didn't put this information: I am not sure when the change was done... I think version 21 but this is a guess, and FossID don't generally mention API changes in their release notes....

As for why this is necessary to drop the support for the older API, I can answer that our support of old versions is a "best effort" and this one in particular would be much effort for few returns...
This property comes in only special cases, for instance here when the scan does not exist.

What is important is to support current version IMO.

Reply:
Ah I found the version in a internal release notes.

Reply:
> I put the version in the commit message.

Great, thank you.

> There is a reason I didn't put this information: I am not sure when the change was done...

If that's the case again in the future. you should explicitly mention that the version that introduced is unknown. That's just honest and better than saying nothing about it.